### ISSUE TITLE
Tab Organiser not being found with search on Chrome Extensions

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#172]
Originally reported on Google Code with ID 173

```
I went to install TO on a friend's Windows PC (I use Linux) but when I searched for
it on the Chrome webstore it is not found!:

https://chrome.google.com/webstore/search-extensions/tab%20organiser

When I got home I looked up this forum from my installed TO - I thought maybe TO had
gone away or something but it still seems like it is being developed.  Does anyone
know why a search does not work?

Thanks,

Phil.
```

Reported by `phil@pricom.com.au` on 2014-06-26 06:03:13

**Comment by Pauan** [comment]
```
Right, I think that's my fault... I think the Chrome Web Store doesn't show version
1 extensions in the search.

A new version of Tab Organizer is in development which will fix that problem, but it
will be quite a while before it's released.

See issue 162 for the new version. You can keep track of progress there.
```

Reported by `pcxunlimited` on 2014-06-27 04:40:13
- **Status changed**: `Assigned`
- **Labels added**: Type-Confirmed
- **Labels removed**: Type-Unconfirmed

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2014-06-27 04:41:29
- **Status changed**: `Duplicate`
- **Merged into**: #162
